packages/jsx-email/src/components/button.tsx

```diff
@@ -83,6 +85,20 @@ export const Button: JsxEmailComponent<ButtonProps> = ({
       {children}
     </a>
   );
+  const vmlFallbackLinkContents = (
+    <a
+      rel={rel}
+      href={href}
+      style={{
+        color: textColor,
+        ...baseStyles,
```

Comment:
probably don't need all the base styles but didn't seem like it would hurt? can pick out the relevant ones though